### PR TITLE
[SU-214] Add feedback links for data table feature previews

### PR DESCRIPTION
**Pull request by nawatts**
Add feedback links to the feature previews page (https://app.terra.bio/#feature-preview).

Screen Shot 2022-09-22 at 3 10 54 PM

And add a banner requesting feedback to the data table. This is only shown when one or both of the relevant feature previews is enabled.

Screen Shot 2022-09-22 at 3 10 48 PM

### CODE REVIEW

File: src/pages/workspaces/workspace/Data.js [hunk 1]
```diff
@@ -422,6 +422,26 @@ const DataTableActions = ({ workspace, tableName, rowCount, entityMetadata, onRe
...
+    isDataTableVersioningEnabled && 'versioning',
+    isDataTableProvenanceEnabled && 'provenance'
+  ]))
+  const feedbackUrl = `mailto:dsp-sue@broadinstitute.org?subject=${encodeURIComponent(`Feedback on data table ${label}`)}`
+
+  return (isDataTableProvenanceEnabled || isDataTableVersioningEnabled) && div({
+    style: {
+      padding: '1rem',
+      borderBottom: `1px solid ${colors.accent()}`,
+      background: colors.accent(0.2),
```
Comment (cahrens) [inline src/pages/workspaces/workspace/Data.js]: This does not have sufficient color contrast.

image

**Review by petesantos — APPROVED** [review]
LGTM with the exception of the a11y issue @cahrens identified

File: src/pages/workspaces/workspace/Data.js (hunk 1 above)
Comment (cahrens) [reply src/pages/workspaces/workspace/Data.js]: Unfortunately this is a general problem on this page with the newly styled elements. Can you please fix all of them?

image

Comment (cahrens) [reply src/pages/workspaces/workspace/Data.js]: I also found another issue on this page that I created a separate ticket for-- https://broadworkbench.atlassian.net/browse/SUP-747.

**Comment by nawatts** [comment]
Changed the banner's background color to white to improve contrast.

Screen Shot 2022-09-23 at 9 52 03 AM

**Review by cahrens — APPROVED** [review]
@nawatts has created https://broadworkbench.atlassian.net/browse/SU-220 to address the other color contrast issues

**Pull request merged**